File: problems/_002/euler_002.js
function fib(n){
  var a = 0;
  var b = 1;
  var temp = 0;
  var sum = 0;
  while (b < n) {
    if ((b % 2) == 0){
      sum += b
    }
    temp = b;
    b = a+b;
    a = temp;
  }
  return sum
}
console.log(fib(4000000));

File: problems/_002/euler_002.go
package main

import "fmt"

func main() {
	a := 0
	b := 1
	temp := 0
	sum := 0
	for b < 4e6 {
		temp = b
		b = b + a
		a = temp
		if b % 2 == 0 { sum += b }
	}
	fmt.Printf("Sum of all even fibonnaci seq terms < 4e6 = %d\n", sum)
}
